nuxeo-mongodb-core/src/main/resources/OSGI-INF/mongodb-core-contrib.xml

```diff
@@ -0,0 +1,24 @@
+<?xml version="1.0"?>
+<component name="org.nuxeo.mongodb.core.MongoDBComponent">
+  <require>org.nuxeo.ecm.core.repository.RepositoryServiceComponent</require>
+  <implementation class="org.nuxeo.mongodb.core.MongoDBComponent" />
+
+  <extension-point name="connection">
+    <documentation>
+      Extension point used to configure access to a MongoDB server or cluster.
+
+      This setup a MongoClient, server are mandatory.
+
+      <code>
+        <connection>
+          <server>mongodb://bob:pass@localhost:27017,otherhost:27018/mydb?replicaSet=test;connectTimeoutMS=300000</server>
+          <dbname>mydb</dbname>
+        </connection>
```

Comment:
As it is there can be only one connection provided by the `MongoDBComponent` in the whole Nuxeo instance. I think this is too restrictive and that you should configure an id for each, `<connection id="default">` for instance, and have the users of the `MongoDBComponent` specify which one they want.